### PR TITLE
[RHACS]: Update Circle CI commands and correct spelling error

### PR DESCRIPTION
**Pull request by tmalove**
Preview link: https://deploy-preview-40345--osdocs.netlify.app/openshift-acs/latest/integration/integrate-with-ci-systems

Fixes: [OSDOCS-3070](https://issues.redhat.com/browse/OSDOCS-3070)
Urgency: Low
Applies to: 3.65 +

NOTE:
- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@tmalove We have received a [LGTM from Evan ](https://issues.redhat.com/browse/OSDOCS-3070?focusedCommentId=19580513&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-19580513) so mybe you can proceed with peer review and merging.

**Comment by tmalove** [comment]
@gaurav-nelson sent for peer review and merge.

File: modules/integrate-circle-ci.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 You can integrate {product-title} with CircleCI.
 
-.Prerequisetes
+.Prerequisites
```
Comment (ctauchen) [inline modules/integrate-circle-ci.adoc:10]: Good spot!

File: modules/integrate-circle-ci.adoc [hunk 2]
```diff
@@ -35,19 +35,25 @@ jobs:
...
+              mkdir junit
       - run:
           name: Install roxctl
           command: |
               curl -H "Authorization: Bearer $ROX_API_TOKEN" https://$STACKROX_CENTRAL_HOST:443/api/cli/download/roxctl-linux -o roxctl && chmod +x ./roxctl
       - run:
           name: Scan images for policy deviations and vulnerabilities
           command: |
-              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_registry/repo/image_name>" <1>
+              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_registry/repo/image_name>" -o junit > junit/image.xml <1>
```
Comment (ctauchen) [inline modules/integrate-circle-ci.adoc:49]: Does the callout need to be changed? Seems like there's new stuff going on here.

File: modules/integrate-circle-ci.adoc [hunk 3]
```diff
@@ -35,19 +35,25 @@ jobs:
...
       - run:
           name: Scan deployment files for policy deviations
           command: |
-              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_deployment_file>" <2>
+              ./roxctl deployment check --endpoint "$STACKROX_CENTRAL_HOST:443" --file "<your_deployment_file>" -o junit > junit/deployment.xml <2>
```
Comment (ctauchen) [inline modules/integrate-circle-ci.adoc:53]: Does the callout need to be changed? Seems like there's new stuff going on here.

**Review by ctauchen — COMMENTED** [review]
A few questions, but otherwise LGTM.

File: modules/integrate-circle-ci.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply modules/integrate-circle-ci.adoc:49]: @ctauchen Thank you for reviewing this. The callouts only refer to the required user input fields, so no change is required as they stay the same.

**Pull request merged**